Taskfile.yml

```diff
@@ -110,10 +110,14 @@ tasks:
     desc: Run the Go-based integration tests
     deps:
       - task: go:build
+        vars:
+          EXTRA_FLAGS: "-covermode=atomic"
     dir: '{{default "./" .GO_MODULE_PATH}}'
     cmds:
       - |
-        go test \
+        rm -fr coverage_data
+        mkdir coverage_data
```

Comment:
is this required?

Reply:
Yes, the directory must exist.
Also every time you run the executable a new file is added to the directory, so if you restart the test you must remove the old data otherwise you get the union of the old coverage with the new one.